tests/test_downloader.py

```diff
@@ -87,29 +88,30 @@ def test_constructor_invalid_dltype(use_env: bool):
         Downloader(**kwargs)
 
 
-@pytest.mark.usefixtures("restore_environ")
+@pytest.mark.reload_modules(morpheus.utils.downloader)
 @pytest.mark.parametrize("dl_method", ["dask", "dask_thread"])
-@mock.patch('dask.config')
+@pytest.mark.usefixtures("reload_modules")
```

Comment:
Remark: Do we need to use both `@pytest.mark.reload_modules` and `@pytest.mark.usefixtures("reload_modules")` I feel like the call to `usefixtures` could be omitted

Reply:
test fails if `usefixtures` is removed. both are used in our other tests.